src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Inputs/ColorPicker/BitColorPickerDemo.razor.scss

```
@@ -1,12 +1,35 @@
-﻿@import "../../../../Styles/abstracts/_functions.scss";
+@import '../../../../Styles/abstracts/_bit-css-variables.scss';
```

Stylelint (17.14.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)